### ISSUE TITLE
Possible Hang

### ISSUE THREAD

**Issue by jesus-mg-ios** [firebase/firebase-ios-sdk#10534]
### Description

Purple advice appears on Xcode when using firebase.

### Reproducing the issue

<img width="793" alt="image" src="https://user-images.githubusercontent.com/85997060/204525910-214d3dfa-e85e-4aa8-8f77-ed351b44f671.png">
<img width="1175" alt="image" src="https://user-images.githubusercontent.com/85997060/204526031-224452c2-34b7-4579-8279-dba321162df8.png">

### Firebase SDK Version

9.6.1

### Xcode Version

14.0.1

### Installation Method

Swift Package Manager

### Firebase Product(s)

All

### Targeted Platforms

iOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by rizafran** [comment]
Hi @jesus-mg-ios, this issue seems related to #10476, and the fix was already included in [v10.2.0](https://firebase.google.com/support/release-notes/ios#version_1020_-_november_15_2022). With this, updating your SDK to the latest version would resolve the issue. Thanks.

**Comment by jesus-mg-ios** [comment]
@rizafran thanks for your reply
